### PR TITLE
Add 7.0, 7.1 to list supported in faq

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/pages/faq.mdx [hunk 1]
```diff
@@ -122,6 +122,8 @@ Teleport provides security-critical support for the current and two previous rel
 
 | Release | Long Term Support | Release Date | Min tsh version |
 | - | - | - | - |
+| 7.1 | No | August 24th, 2021 | 6.0.0 |
+| 7 | Yes | August 10th, 2021 | 6.0.0 |
```
Comment (r0mant) [inline docs/pages/faq.mdx:126]: Given that we release minor versions more often now, this table will grow huge pretty quick. Maybe just keep `7.x` or `7` in here? I'm also not sure `7` (as in 7.0) is considered LTS. @russjones What do you think?

Comment (russjones) [reply docs/pages/faq.mdx:126]: Yeah, the support is for major versions now. So Teleport 7, 6, and 5. For example, any security patches will be off of 7.2 now and not off 7.0 or 7.1.

Comment (stevenGravy) [reply docs/pages/faq.mdx:126]: @russjones do you want to keep this format or change? I just followed the earlier ones.

Comment (webvictim) [reply docs/pages/faq.mdx:126]: I think we should change the language above this on the page to be explicit that it's the current major version plus two back which are supported.

We should also add that 4.4 is EOL.

Comment (klizhentas) [reply docs/pages/faq.mdx:126]: @stevenGravy @r0mant let's use `7.x.x` format instead of `7.1` and `7`

**Comment by russjones** [comment]
@stevenGravy Can you update this and then let's merge this in.

**Comment by russjones** [comment]
Closing, I've fixed this in the 8.0 PR.

**Pull request closed without merging**